### ISSUE TITLE
IIS installation error

### ISSUE THREAD

**Issue by vermokhin** [chef-cookbooks/iis#317]
### Cookbook version
5.1.0

### Chef-client version
Chef 12.12.15

### Platform Details
Windows Server 2008 R2 Standart

### Scenario:
Install IIS

### Expected Result:
IIS will install propery and complete

### Actual Result:

```
[31mError executing action `install` on resource 'windows_feature[IIS-WebServerRole]'[0m
build	24-Mar-2017 07:31:06	================================================================================[0m
build	24-Mar-2017 07:31:06	
build	24-Mar-2017 07:31:06	[0mNoMethodError[0m
build	24-Mar-2017 07:31:06	-------------[0m
build	24-Mar-2017 07:31:06	windows_feature_dism[IIS-WebServerRole] (c:/chef/cache/cookbooks/windows/resources/feature.rb line 44) had an error: NoMethodError: undefined method `feature' for Custom resource windows_feature_dism from cookbook windows[0m
build	24-Mar-2017 07:31:06	
build	24-Mar-2017 07:31:06	[0mCookbook Trace:[0m
build	24-Mar-2017 07:31:06	---------------[0m
build	24-Mar-2017 07:31:06	c:/chef/cache/cookbooks/windows/resources/feature_dism.rb:77:in `block in class_from_file'
build	24-Mar-2017 07:31:06	[0mc:/chef/cache/cookbooks/compat_resource/files/lib/chef_compat/monkeypatches/chef/runner.rb:78:in `run_action'
build	24-Mar-2017 07:31:06	[0mc:/chef/cache/cookbooks/compat_resource/files/lib/chef_compat/monkeypatches/chef/runner.rb:106:in `block (2 levels) in converge'
build	24-Mar-2017 07:31:06	[0mc:/chef/cache/cookbooks/compat_resource/files/lib/chef_compat/monkeypatches/chef/runner.rb:106:in `each'
build	24-Mar-2017 07:31:06	[0mc:/chef/cache/cookbooks/compat_resource/files/lib/chef_compat/monkeypatches/chef/runner.rb:106:in `block in converge'
build	24-Mar-2017 07:31:06	[0mc:/chef/cache/cookbooks/compat_resource/files/lib/chef_compat/monkeypatches/chef/runner.rb:105:in `converge'
build	24-Mar-2017 07:31:06	[0mc:/chef/cache/cookbooks/compat_resource/files/lib/chef_compat/monkeypatches/chef/runner.rb:78:in `run_action'
build	24-Mar-2017 07:31:06	[0mc:/chef/cache/cookbooks/compat_resource/files/lib/chef_compat/monkeypatches/chef/runner.rb:106:in `block (2 levels) in converge'
build	24-Mar-2017 07:31:06	[0mc:/chef/cache/cookbooks/compat_resource/files/lib/chef_compat/monkeypatches/chef/runner.rb:106:in `each'
build	24-Mar-2017 07:31:06	[0mc:/chef/cache/cookbooks/compat_resource/files/lib/chef_compat/monkeypatches/chef/runner.rb:106:in `block in converge'
build	24-Mar-2017 07:31:07	[0mc:/chef/cache/cookbooks/compat_resource/files/lib/chef_compat/monkeypatches/chef/runner.rb:105:in `converge'[0m
build	24-Mar-2017 07:31:07	
build	24-Mar-2017 07:31:07	[0mResource Declaration:[0m
build	24-Mar-2017 07:31:07	---------------------[0m
build	24-Mar-2017 07:31:07	# In c:/chef/cache/cookbooks/iis/recipes/default.rb
build	24-Mar-2017 07:31:07	[0m
build	24-Mar-2017 07:31:07	[0m 25:   windows_feature feature do
build	24-Mar-2017 07:31:07	[0m 26:     action :install
build	24-Mar-2017 07:31:07	[0m 27:     all !Opscode::IIS::Helper.older_than_windows2012?
build	24-Mar-2017 07:31:07	[0m 28:     source node['iis']['source'] unless node['iis']['source'].nil?
build	24-Mar-2017 07:31:07	[0m 29:   end
build	24-Mar-2017 07:31:07	[0m 30: end
build	24-Mar-2017 07:31:07	[0m
build	24-Mar-2017 07:31:07	[0mCompiled Resource:[0m
build	24-Mar-2017 07:31:07	------------------[0m
build	24-Mar-2017 07:31:07	# Declared in c:/chef/cache/cookbooks/iis/recipes/default.rb:25:in `block in from_file'
build	24-Mar-2017 07:31:07	[0m
build	24-Mar-2017 07:31:07	[0mwindows_feature("IIS-WebServerRole") do
build	24-Mar-2017 07:31:07	[0m  action [:install]
build	24-Mar-2017 07:31:07	[0m  updated true
build	24-Mar-2017 07:31:07	[0m  updated_by_last_action true
build	24-Mar-2017 07:31:07	[0m  retries 0
build	24-Mar-2017 07:31:07	[0m  retry_delay 2
build	24-Mar-2017 07:31:07	[0m  default_guard_interpreter :default
build	24-Mar-2017 07:31:07	[0m  declared_type :windows_feature
build	24-Mar-2017 07:31:07	[0m  cookbook_name "iis"
build	24-Mar-2017 07:31:07	[0m  recipe_name "default"
build	24-Mar-2017 07:31:07	[0m  feature_name "IIS-WebServerRole"
build	24-Mar-2017 07:31:07	[0mend
build	24-Mar-2017 07:31:07	[0m
build	24-Mar-2017 07:31:07	[0mPlatform:[0m
build	24-Mar-2017 07:31:07	---------[0m
build	24-Mar-2017 07:31:07	i386-mingw32[0m
```

**Comment by vermokhin** [comment]
Issue was resolved ater windows cookbook version 3.0.3 released

**Comment by cheeseplus** [comment]
This is a bug in the Windows cookbook that was fixed @ https://github.com/chef-cookbooks/windows/commit/cf99c8a92a0298899b23363163c9cb92cc2cafa8. Version 3.0.3 has been released to include this fix.
